core/renderers/geras/path_object.js

```diff
@@ -66,7 +67,40 @@ Blockly.geras.PathObject = function(root) {
    */
   this.svgPathLight = Blockly.utils.dom.createSvgElement('path',
       {'class': 'blocklyPathLight'}, this.svgRoot);
+
+
+  /**
+   * Primary colour of the block in '#RRGGBB' format.
+   * @type {string}
+   * @package
+   */
+  this.primaryColour = '#000000';
```

Comment:
Why do you need to re-declare these here? If they're inherited.

Reply:
I'm not calling the superclass constructor here, because the order of creating the svg path objects matters.

The old pattern I would use puts these declarations outside of the constructor--they would be on the prototype and left below.  We agreed to stop doing that, but it seems relevant here.